Tests/ShapedArrayTests/ShapedArrayTests.swift

```diff
@@ -8,4 +9,89 @@ final class ShapedArrayTests: XCTestCase {
         // results.
         print(ShapedArray(shape: [4], scalars: [1, 2, 3, 4]))
     }
+    
+    func testElementIndexing() {
+        let array3D = ShapedArray<Float>(shape: [3, 4, 5], scalars: Array(stride(from: 0.0, to: 60, by: 1)))
```

Comment:
Why use both Float and Integer notation when what we want are Integers? I.e. why `0.0` and `60` (no `.0`)?

Reply:
I see that it's inconsistent here, but I'd actually prefer to use the short notation for numbers where possible. Since it's just shorter.  What is your opinion? (so 0, 60 and 1 in this case)

Reply:
Why are you importing CoreML? Please remove - it only exists on Apple platforms.

Reply:
In terms of notation, I don’t have a real preference, but I do believe that when one can be explicit with low or reasonable effort it’s a good thing to do because it lessons confusion / increases readability.

Reply:
> Why are you importing CoreML? Please remove - it only exists on Apple platforms.

Whoops think that was some autoimport by Xcode 😅